packages/forma-36-react-components/src/components/Notification/Notification.stories.tsx

```diff
@@ -1,11 +1,8 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
-import { button, text, number, boolean, select } from '@storybook/addon-knobs';
 
 import Button from '../Button';
-import Notification from './index';
-import notes from './Notification.md';
+import Notification, { NotificationsAPI } from './index';
```

Comment:
I know it's not a change that you made but... Could you rename NotificationAPI to NotificationProps in this PR, pls?
Just so it follows the naming convention we are using in the other components 

🙏

Reply:
@gui-santos as we export this name (most likely is not used anywhere), would it be in theory, a breaking change if we change the namespace?

Reply:
I think it's only used here, but I see your point 👍 
let's do it later then